using-mobile-extensions/offer-decisioning/README.md

```diff
@@ -9,6 +9,9 @@
 * Adobe Experience Platform Core and Edge mobile extensions have been implemented in the app
 * Launch UI config is published
 
+## Limitations
+Some Offer constraints are currently not supported with the mobile Experience Edge workflows, for example `Capping`. Capping field value specifies the number of times an offer can be presented across all users. For more details, see [Offer eligibility rules and constraints](https://experienceleague.adobe.com/docs/offer-decisioning/using/managing-offers-in-the-offer-library/creating-personalized-offers.html#eligibility).
```

Comment:
Is there a reason that "Offer" is capitalized here?

Reply:
Also, I'd go with "The capping field value specifies..." since, at least to me, the way it's currently phrased sounds like we're trying to cap field values. Adding the "the" in front removes that ambiguity to me.

Reply:
Lastly, "For more details, see the [offer eligibility rules and constraints guide\]\(link)."

Reply:
Yes, in my opinion `Offer` should be capitalized because we are referring to a specific AEP component. You can see mixed usage in the link shared above but in our docs, it will help identify the product when you read the sentence.

Reply:
I can add `The...` in the beginning but the field name is `Capping` so I believe we should retain the case.

Reply:
I talked to the doc writer for Decisioning, and he confirmed to me that it should be a lowercase "o" for "offer".

Reply:
Also, maybe we can do "The `Capping` field then? That would make it super duper clear what we're talking about there.

Reply:
> I talked to the doc writer for Decisioning, and he confirmed to me that it should be a lowercase "o" for "offer".

Sounds good.

Reply:
I've updated the pr as per additional review comments @calebk1